### PR TITLE
change Targets date property default to "now" #516

### PR DESCRIPTION
**Pull request by elvisdorkenoo**
The original issue is [Target's date property should default to "now"#516](https://github.com/medic/cht-conf/issues/516).
This PR update the documentation after this [PR](https://github.com/medic/cht-conf/pull/514) got approved.
Now that the Target's date property default to 'now', we are updating the docs accordingly.

### CODE REVIEW

File: content/en/apps/reference/targets.md [hunk 1]
```diff
@@ -20,30 +20,30 @@ All targets are defined in the `targets.js` file as an array of objects accordin
...
-| `passesIf` | `function(contact, report)` | For `type: 'percent'`, return true to increment the numerator. | yes, if `type: 'percent'`. Forbidden when `groupBy` is defined. |
-| `date` | `'reported'` or `'now'` or `function(contact, report)` | When `'reported'`, the target will count documents with a `reported_date` within the current month. When `'now'`, target includes all documents. A function can be used to indicate when the document should be counted. Default is `'reported'`. | no |
+| property | type | description                                                                                                                                                                                                                                                                                                                                                                                                                   | required |
```
Comment (latin-panda) [inline content/en/apps/reference/targets.md:23]: Please remove the other changes, I mean the weird spaces, it makes the code hard to read. 
Screen Shot 2022-12-05 at 1 10 47 pm

Please just leave the relevant change for this ticket.

File: content/en/apps/reference/targets.md [hunk 2]
```diff
@@ -20,30 +20,30 @@ All targets are defined in the `targets.js` file as an array of objects accordin
...
+| `passesIf` | `function(contact, report)` | For `type: 'percent'`, return true to increment the numerator.                                                                                                                                                                                                                                                                                                                                                                | yes, if `type: 'percent'`. Forbidden when `groupBy` is defined. |
+| `date` | `'reported'` or `'now'` or `function(contact, report)` | When `'reported'`, the target will count documents with a `reported_date` within the current month. When `'now'`, target includes all documents. A function can be used to indicate when the document should be counted. Default is `'now'`.                                                                                                                                                                                  | no |
```
Comment (latin-panda) [inline content/en/apps/reference/targets.md:37]: You can also put something like this to add more clarity about what happens when it's a function:
 

> When this property is undefined or the value is null the default is `'now'`.

**Review by latin-panda — CHANGES_REQUESTED** [review]
Thanks Elvis,
I left a couple of comments below. We try as much as possible to ship PRs of cht-docs when the ticket goes to AT (not when it passed AT) because often times QA also reads the docs to check things. In this particular case, when I chatted with QA about this ticket, I made the clarification when reading the docs. 

Did you coordinated with @esthermmoturi as discussed last week to clarify the targets's owner with a better example? 
That is a quick win that we can include here and with a nice impact for others.

File: content/en/apps/reference/targets.md (hunk 1 above)
Comment (esthermmoturi) [reply content/en/apps/reference/targets.md:23]: @latin-panda yes I have a PR with the changes on task-owner vs requester I am waiting to merge the changes

Comment (latin-panda) [reply content/en/apps/reference/targets.md:23]: Awesome thanks @esthermmoturi! 🤩

Comment (elvisdorkenoo) [reply content/en/apps/reference/targets.md]: Yes thanks Esther, about the weird spaces, it is something that my IDE was doing when showing the preview besides, I managed to fix, thanks for noticing.

File: content/en/apps/reference/targets.md (hunk 2 above)
Comment (elvisdorkenoo) [reply content/en/apps/reference/targets.md]: sure !

File: content/en/apps/reference/targets.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ All targets are defined in the `targets.js` file as an array of objects accordin
 
 | property | type | description | required |
 |---|---|---|---|
-| `id` | `string` | An identifier for the target. | yes, unique |
+| `id` | `string` | An identfier for the target. | yes, unique |
```
Comment (latin-panda) [inline content/en/apps/reference/targets.md:25]: typo
```suggestion
| `id` | `string` | An identifier for the target. | yes, unique |
```

**Review by latin-panda — CHANGES_REQUESTED** [review]
Almost! Just a tiny typo

**Review by latin-panda — APPROVED** [review]
Nice!!

**Pull request merged**